scripts/measures/2018/import-quality-measures.js

```diff
@@ -58,12 +58,12 @@ const MAIN_FIELDS = {
 
 // Source CSV column names below are mapped to their measures data names
 const SUBMISSION_METHODS = {
-  claimsMethod: 'claims',
-  certifiedSurveyVendorMethod: 'certifiedSurveyVendor',
-  electronicHealthRecordMethod: 'electronicHealthRecord',
-  cmsWebInterfaceMethod: 'cmsWebInterface',
-  administrativeClaimsMethod: 'administrativeClaims',
-  registryMethod: 'registry'
+  methodsClaims: 'claims',
```

Comment:
they changed this a while back in the CSV they sent us but I hadn't updated it in the code. looks like a permanent change so i'll update it now